packages/scripts/src/application.ts

```diff
@@ -672,6 +754,37 @@ export class Application {
     }
 }
 
+function getFilePathInPackage(
+    fs: IFileSystemSync,
+    packageName: string,
+    context: string,
+    filePath: string,
+    isRoot: boolean
+) {
+    const relativeFilePath = fs.relative(context, filePath);
+    const relativeRequest = fs
+        .join(fs.dirname(relativeFilePath), fs.basename(relativeFilePath, fs.extname(relativeFilePath)))
+        .replace(/\\/g, '/');
+    return isRoot
```

Comment:
I'm a bit confused. In a mono-repo with:
```
/package.json
/packages/feature-a/package.json
/packages/feature-b/package.json
```
if I'm building `feature-a` into `/packages/feature-a/dist`, the manifest inside that directory should use relative paths for any file inside the `feature-a` package, and package requests for any file outside that package. Is that what's happening here?

Would `isRoot` be true just for `feature-a`? If so, would it because engine build is executed inside `/packages/feature-a` directory?

Reply:
yes and yes.
meaning that if running in monorepo, and running the command from the root, all features found in packages of the monorepo will be isRoot = true. 
I think, that maybe a better approach would be that if featureName was passed, isRoot = true only for the feature being built, otherwise isRoot = false for all.

What do you think?

Reply:
That depends on what else references `isRoot`, and what assumptions were made there.

There's also the case, where I build `feature-a` (example above) into `/outside-packages/some/elsewhere`, the relative references are no longer the correct approach.

So it's a matter of current context and target directory, and we currently assume `engine build` will:
- be executed within the package you're building.
- output into a folder within that package.

I'm fine with those limitations for now, but anyone trying to go out of the expected assumtions may get into trouble.